FSharp.AutoComplete/Program.fs

```diff
@@ -366,7 +370,7 @@ module internal Main =
      new RangeConverter() :> JsonConverter
     |]
 
-  let prAsJson o = printAgent.WriteLine (JsonConvert.SerializeObject(o, jsonConverters))
+  let prAsJson o = printAgent.WriteLine (JsonConvert.SerializeObject(o, Formatting.Indented,  jsonConverters))
```

Comment:
I'm afraid not really. The JSON parsers that interpret what is printed by FSAC assume that each message is exactly one line.